spec/controllers/gift_exchange_controller_spec.rb

```diff
@@ -16,12 +17,8 @@
       expect(@collection.reload.challenge_type).to eq(nil)
     end
 
-    it "sets a flash message" do
-      expect(flash[:notice]).to eq("Challenge settings were deleted.")
-    end
-
     it "redirects to the collection's main page" do
```

Comment:
This should be updated to "redirects to the collection's main page with a notice" or similar now that you've combined this example and the previous one.